src/conf_world/lib/model/conference_model.dart

```diff
@@ -41,4 +54,18 @@ class ConferenceInfo {
         'saved': saved,
       };
 
+  getIconColor() {
```

Comment:
parameter saved should be boolean

Reply:
It cant be a bollean because sqlite can't save true or false. So we use 1 or 0 for true or false respectively